controllers/records/config_list.yaml

```diff
@@ -37,6 +37,11 @@ showSorting: true
 #     column: created_at
 #     direction: desc
 
+replication:
+    enabled: true
+    allowed: [name, status]
+    ignored: []
```

Comment:
In this case it would be more useful to set the ignored property to just status as there are a lot of test fields in the Record model.